docs/ref/models/fields.txt

```diff
@@ -1810,10 +1810,16 @@ relation. These attribute are present on all fields; however, they will only
 have meaningful values if the field is a relation type
 (:attr:`Field.is_relation=True <Field.is_relation>`).
 
+.. attribute:: Field.many_to_one
```

Comment:
Actually should I move many_to_many to the top? Then it'd be effectively alphabetical.